README.md

```diff
@@ -29,126 +29,177 @@
+    close()
+        Returns true ... ok (1ms)
+
+test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out (4ms)
 ```
 
 ## Features
 
-- Hooks
-  - `beforeEach`
-  - `beforeAll`
-  - `afterEach`
-  - `afterAll`
 - Descriptive naming for your tests
 - Lightweight
 - Zero dependencies
 - Simple and easy to use
 - Asynchronous support
 - Still uses `Deno.test` under the hood
+- Skip functionality
+- Mock requests
 
 ## Documentation
-ADD DOCUMENTATION ABOUT: HOOKS, IGNORING
 
-### `Rhum.TestPlan`
+### `Rhum.testPlan`
 
-Groups up test suites to describe a test plan. Usually, a test plan is per file and contains the tests and suites for a single file
+Groups up test suites to describe a test plan. Usually, a test plan is per file and contains the tests suites and test cases for a single file.
 
 ```typescript
-Rhum.TestPlan("app_test.ts", () => {
-  Rhum.TestSuite("run()", () => {
+Rhum.testPlan("app_test.ts", () => {
+    ...
     ...
-  })
-})
+    ...
+});
 ```
 
-### `Rhum.TestSuite`
+### `Rhum.testSuite`
 
-A test suite usually describes a method or property name, and groups up all test cases for that method or property. There can be as many test suites under a test plan as you want.
+A test suite usually describes a method or property name and groups up all test cases for that method or property. You can define multiple test suites under a test plan.
 
 ```typescript
-Rhum.TestPlan("app_test.ts", () => {
-  Rhum.TestSuite("run()", () => {
+Rhum.testPlan("app_test.ts", () => {
+  Rhum.testSuite("run()", () => {
+    ...
+    ...
     ...
-  })
-  Rhum.TestSuite("close()", () => {
+  });
+  Rhum.testSuite("close()", () => {
     ...
-  })
-})
+    ...
+    ...
+  });
+});
+```
+
+### `Rhum.testCase`
+
+A test case is grouped by a test suite and it is what makes the assertions - it is the test. You can define multiple test cases under a test suite. Test cases can also be asynchronous.
+
+```typescript
+Rhum.testPlan("app_test.ts", () => {
+  Rhum.testSuite("run()", () => {
+    Rhum.testCase("should return true", () => {
+      Rhum.assert.assertEquals(run(), true);
+    });
+    Rhum.testCase("should return false", () => {
+      Rhum.assert.assertEquals(run(), false);
+    });
+  });
+});
 ```
 
-### `Rhum.TestCase`
+### `Rhum.asserts`
 
-A test case is grouped by a test suite and it is what makes the assertions - it is the test. A suite can have as many test cases as you want. Test cases can also be asynchronous.
+The [asserts](https://deno.land/std/testing/asserts.ts) module, but attached to `Rhum`.
 
 ```typescript
-Rhum.TestPlan("app_test.ts", () => {
-  Rhum.TestSuite("run()", () => {
-    Rhum.TestCase("Returns true", () => {
-      Rhum.assertEquals(run(), true);
-    })
-    Rhum.TestCase("Returns false", () => {
-      Rhum.assertEquals(run(), false);
-    })
-  })
-})
+Rhum.asserts.assertEquals(true, true) // pass
+Rhum.asserts.assertEquals(true, false) // fail
 ```
 
-### `Rhum.assertEquals`
+### `Rhum.skip`

```

Comment:
oh this is interesting. what's a use case for skipping a written test?

Reply:
In case you want to ignore it temporarily

Reply:
We’re following this API: heres the issue: https://github.com/denoland/deno/issues/4092

Reply:
There were times where tests failed in windows and we had to comment out the tests. This is the same approach but with a method.